plugins/inputs/elasticsearch_query/elasticsearch_query.go

```diff
@@ -242,8 +235,10 @@ func (e *ElasticsearchQuery) esAggregationQuery(acc telegraf.Accumulator, aggreg
 func init() {
 	inputs.Add("elasticsearch_query", func() telegraf.Input {
 		return &ElasticsearchQuery{
-			Timeout:             config.Duration(time.Second * 5),
 			HealthCheckInterval: config.Duration(time.Second * 10),
+			HTTPClientConfig: httpconfig.HTTPClientConfig{
+				ResponseHeaderTimeout: config.Duration(5 * time.Second),
```

Comment:
Any reason you are _only_ setting the `ResponseHeaderTimeout` and not the `Timeout` as it was done before?

Reply:
I saw that 5 seconds is already the default Timeout in HTTPClientConfig struct (https://github.com/influxdata/telegraf/blob/master/plugins/common/http/config.go#L59). Do you think it would make more sense to set 5 seconds at the plugin level to make sure this is not changed?

Reply:
Yeah that would be good. I only noticed because in the tests you set the timeout...

Reply:
ok. It should be fixed with my last commit